### DISCUSSION TITLE
GraphRAG in Open WebUI with local LLMs

### DISCUSSION THREAD

**Discussion by win4r** [microsoft/graphrag#576] [Ideas]
###This is my project:https://github.com/win4r/GraphRAG4OpenWebUI 

Hey folks! I've got something exciting to share with you all. I've taken Microsoft's awesome GraphRAG technology and turned it into an API that plugs right into Open WebUI. It's like giving your web interface a supercharged brain for information retrieval.
I've built this cool bridge between cutting-edge research and practical applications. Now you can harness the power of graph-based retrieval-augmented generation in your Open WebUI projects with just a few simple API calls. It's all about making advanced search capabilities more accessible and easier to integrate.
Whether you're into local searches, global queries, or want to tap into the Tavily search engine, this API has got you covered. And the best part? It now supports local LLMs and embedding models too, so you can keep things private and cost-effective.
Can't wait to see what you all build with this! Let me know if you have any questions or ideas to make it even better. Happy coding, everyone!"
How's that? This introduction conveys your excitement about the project, highlights its key features, and invites engagement from the community. Would you like me to modify anything in this introduction?

**Comment by flefevre** [comment]
Any idea on what is the agenda to integrate such strategy?

**Comment by BigBroData** [comment]
I found your idea is practical good. Can you help me with running in Open WebUI?

**Comment by kityr** [comment]
![image](https://github.com/user-attachments/assets/9c88295c-3573-4f03-977c-f8405b286057)
bro used an llm to present his project and just copied and pasted without looking 💀

**Reply by schauerstoff** [reply]
I kinda felt like it was AI at "I've built this cool bridge .." - no shade though, I also ask AI to help phrase things ;;

**Comment by gilbrotheraway** [comment]
does it show the graphs in the actual UI?

**Comment by ntsarb** [comment]
Which version of GraphRAG does this support?

**Comment by gilbrotheraway** [comment]
graphiti is way better

**Reply by flefevre** [reply]
https://github.com/getzep/graphiti
